docker-compose.yml

```diff
@@ -1,4 +1,14 @@
 services:
+  db:
```

Comment:
It might be good to give this a `container_name` value as well, so it doesn't just get named 'postgres'. This will make debugging easier. I'd recommend `inatator_db` or something similar.